incident.go

```diff
@@ -653,4 +653,57 @@ func (c *Client) ManageIncidentAlertsWithContext(ctx context.Context, incidentID
 	return &result, nil
 }
 
-/* TODO: Create Status Updates */
+// IncidentStatusUpdate is a status update for the specified incident.
+type IncidentStatusUpdate struct {
+	ID        string    `json:"id,omitempty"`
+	Message   string    `json:"message,omitempty"`
+	CreatedAt string    `json:"created_at,omitempty"`
+	Sender    APIObject `json:"sender,omitempty"`
+}
+
+// CreateIncidentStatusUpdateResponse is returned from the API as a response to creating an incident status update.
+type CreateIncidentStatusUpdateResponse struct {
```

Comment:
Since we only use this to deserialize the JSON response, and don't actually return it to consumers (or accept it in any methods), does this need to be exported?